apps/web-giddh/src/app/proforma-invoice/proforma-invoice.component.ts

```diff
@@ -3985,6 +3985,21 @@ export class ProformaInvoiceComponent implements OnInit, OnDestroy, AfterViewIni
         }
     }
 
+    /**
+     * Quantity field blur handler
+     *
+     * @param {SalesEntryClass} entry Current entry
+     * @param {SalesTransactionItemClass} transaction Current transaction
+     * @memberof ProformaInvoiceComponent
+     */
+    public handleQuantityBlur(entry: SalesEntryClass, transaction: SalesTransactionItemClass): void {
+        if (transaction.quantity !== undefined) {
```

Comment:
can we handle null value here?

Reply:
No because we need to set the quantity to 0 if user leaves the field empty which is null and` Number(null) `is 0 therefore only `undefined` is handled on purpose